Docs: Removed condescending language (#9040)
facebook/jest: docs/TutorialAsync.md

## `async`/`await`

> Writing tests using the `async`/`await` syntax is less complicated than you might expect. Here is how you'd write the same examples from before:

Reviewer: ```suggestion
Writing tests using the `async`/`await` syntax is also possible. Here is how you'd write the same examples from before:
```
